docs/contributing/minutes.md

```diff
@@ -4,6 +4,73 @@ title: Team Minutes
 sidebar_label: Team Minutes
 ---
 
+## 03-02-2022
+
+Attendees: Ali, Greg, Sally, Daniel, Mitch
+
+### Agenda
+
+- Celebrating fundamentals
+- Welcoming Sanyia
+- Sprint planning
+- Plan for 2022
+
+### Celebrating fundamentals
+
+- Woop! 🎉 Go Fundamentals - now published!
+- Fundamentals training starting on 12th Match for S African volunteers
+- Fundamentals starts in April with South Africa Class 2
+
+### Plan 2022
+
+- Improve student tracking
+- Further delegation to module teams
+- Cross module stuff
+- Problem solving
+- Integration with PD
+- Working on students with code reviews
+- Module contracts - Well defined prequisites and outcomes for any module on the course
+-
+- Iterate on module template for Code Your Future
+- Succession planning: learning how to hand things over in a sustainable way 🤔
+
+## Module teams & expectations
+
+- Work needed to ensure that module teams are meeting regularly
+- Getting alums to attend syllabus meetings - really valued input as they've just done the courses
+- Make teams more accountable on Slack - have bots that remind about meetings in public channels ?
+- Someone could join JS2 to suppor them with getting through some tasks
+- Do we need syllabus team folks to join the module teams to get them going ( time limited )
+
+### Welcoming Sanyia
+
+- We welcome to Sanyia to the Syllabus Team
+- Software Engineer at Deliveroo Started at CYF 2 years ago
+- Some node module changes proposals ( Barath / Sanyia ) - Use of wordle in Lesson 3 of the Node module
```

Comment:
@sansaid Please could you add a link to your node module proposals - I think I saw a google doc in the meeting